misc/python/materialize/mzcompose/composition.py

```diff
@@ -775,7 +778,7 @@ def run(
         return self.invoke(
             "run",
             *(["--entrypoint", entrypoint] if entrypoint else []),
-            *(f"-e{k}={v}" for k, v in env_extra.items()),
```

Comment:
We should consider printing `env_extra` to make it easier to debug. Including the values in the command spams the terminal, so I changed it to just pass the keys here, and pass the environment separately.